docs/manuals/core/versioning/production_mode.md

```diff
@@ -70,6 +70,118 @@ the latest is used and converted to production before running.
 Note that once the version is converted to production, the application can access all production
 entities, including from older production versions.
 
+# Change config of an existing production version
+
+Similar to experiment mode, to re-run an existing production version, the configuration
+of the application must be the same. There must be no breaking change.
+
+Let's assume multiple changes have been made to the configuration in `main.py`.
+
+```python linenums="1" title="main.py"
+{%
+include-markdown "./code_example/main_with_multiple_changes.py"
+comments=false
+%}
+```
+
+```console
+$ python main.py --production 2.0
+[2023-01-25 12:52:05,484][Taipy][ERROR] The version 2.0 Configuration is conflicted with the current Configuration:
+    DATA_NODE "input" has attribute "path" added: input.pkl
+    DATA_NODE "output" has attribute "path" added: output.pkl
+    DATA_NODE "input" has attribute "scope" modified: SCENARIO:SCOPE -> GLOBAL:SCOPE
+    DATA_NODE "output" has attribute "scope" modified: SCENARIO:SCOPE -> GLOBAL:SCOPE
+    TASK "my_print_algo" has attribute "skippable" modified: False:bool -> True:bool
+
+Please add a new production version with migration functions or run your application with --taipy-force option to override the Config of production version 2.0.
```

Comment:
I believe we should change this message. It is not dissuasive enough to force a run. Something like:

"Please add a new production version with migration functions.
You can also run your application with `--taipy-force` option to override the Config of production version 2.0. Nevertheless, be cautious. You must ensure the entities created from a previous Config remains compatible with the new Config."

What do you think?

Of course this is not on taipy-doc...

Reply:
I think it is more clear but too wordy. Not sure though

Reply:
What do you think about improving the documentation of the `--taipy-force`?

I think it is better than explain it here.

Reply:
Independently from improving `--taipy-force` option documentation, I believe it's important to let the user know in the logs that it is not just an alternative to implement a migration function. Some prerequisites must be ensured before using `--taipy-force`.
Maybe the following message is simpler than the previous one.

"Please add a new production version with migration functions.
If old entities are ensured to remain compatible with the new Config, you can also run your application with `--taipy-force` option to override the Config of production version 2.0."

Reply:
I think the new message is much more reasonable. Thank you @jrobinAV.

I will modify both the doc and the code.